pkg/kafka/kafka-publisher.go

```diff
@@ -160,21 +161,26 @@ func NewKafkaPublisher(kafkaSrv string) (pub.Publisher, error) {
 	config.Admin.Retry.Max = 100
 	config.Version = sarama.V1_1_0_0
 
-	br := sarama.NewBroker(kafkaSrv)
+	kafkaSrvs := strings.Split(kafkaSrv, ",")
+	ca, err := sarama.NewClusterAdmin(kafkaSrvs, config)
+	if err != nil {
+		glog.Errorf("failed to create cluster admin: %+v", err)
```

Comment:
Are you sure it is safe to just print error if NewClusterAdmin fails? I see ca is used as a parameter in waitForControllerBrokerConnection.

Reply:
fixed in https://github.com/sbezverk/gobmp/pull/236/commits/3c7fbf3dbe99ec0ef3981dcc8634c1931cb1c054